cookbook/security/entity_provider.rst

```diff
@@ -583,6 +583,11 @@ The code below shows the implementation of the
         }
     }
 
+.. tip::
+
+    Don't forget to add the repository class to the 
```

Comment:
Agreed, anyone interested in the background will find the needed information in the referenced chapter.